ModuleNotFoundError: No module named 'metpy'

Our automated tests recently started failing with an `ModuleNotFoundError`:

```
tsdat/io/converters.py:6: in <module>
    import act  # type: ignore
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/act/__init__.py:17: in <module>
    from .qc import QCFilter, QCTests, clean
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/lazy_loader/__init__.py:76: in __getattr__
    submod = importlib.import_module(submod_path)
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/importlib/__init__.py:1[27](https://github.com/tsdat/tsdat/actions/runs/3598848376/jobs/6061971915#step:5:28): in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/act/qc/qcfilter.py:12: in <module>
    from act.qc import comparison_tests, qctests, bsrn_tests
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/lazy_loader/__init__.py:73: in __getattr__
    return importlib.import_module(f"{package_name}.{name}")
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/act/qc/comparison_tests.py:11: in <module>
    from act.utils.data_utils import convert_units
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/act/utils/data_utils.py:8: in <module>
    import metpy
E   ModuleNotFoundError: No module named 'metpy'
```

`act` version 1.3.3 for the github MacOS and Ubuntu runners for python 3.8, 3.9, and 3.10

Just adding metpy to my `requirements.txt` file fixes this, but I thought you all should know.

It looks like the offending code in `act/utils/data_utils` was added in release 1.3.1. I would recommend adding metpy as a required package or refactoring how it gets imported in that file so that it doesn't throw an error unless it would actually be used.

--- a/requirements.txt
+++ b/requirements.txt
@@ -15,5 +15,5 @@ cftime
 netcdf4
 lazy_loader
 fsspec
-lazy_loader
+metpy
 lxml

